testfn.go

```diff
@@ -148,6 +148,11 @@ func (t *testcmd) test(c *cli.Context) error {
 
 func runlocaltest(target string, in *inputMap, expectedOut *outputMap, expectedErr *string, env map[string]string) error {
 	inBytes, _ := json.Marshal(in.Body)
+	if string(inBytes) == "\"\"" {
```

Comment:
Should be checking the error returned from marshal.

Reply:
i'll add check for this, not sure why it's being skipped.